app-misc/boxes/boxes-2.2.0.ebuild

```diff
@@ -11,11 +11,15 @@ SRC_URI="https://github.com/ascii-boxes/boxes/archive/v${PV}.tar.gz -> ${P}.tar.
 
 LICENSE="GPL-2"
 SLOT="0"
-KEYWORDS="~alpha amd64 ~ia64 ~mips ppc ppc64 sparc x86"
+KEYWORDS="~alpha ~amd64 ~ia64 ~mips ~ppc ~ppc64 ~sparc ~x86"
 
 DEPEND="
 	sys-devel/flex
-	sys-devel/bison"
+	sys-devel/bison
+	dev-libs/libpcre2[pcre32]
```

Comment:
libpcre2 will belong in DEPEND + RDEPEND as it's a library that installed executables are linked against